### PR TITLE
02 git rock paper scissors

### PR DESCRIPTION
**Pull request by SergeyChekanov**
Develope game rock_paper_scissors, solution of my home work in  gl-kernel-training-2019.
Please review with my solution and if everything ok, allow me merging branches.

### CODE REVIEW

File: 02-Git/RockPaperScissors.sln [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+		{54392AF4-6C91-468F-B7A1-474D22EA0734}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{54392AF4-6C91-468F-B7A1-474D22EA0734}.Release|Any CPU.Build.0 = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		SolutionGuid = {10C356ED-1781-4B52-9CFE-CBBBABC7CB89}
+	EndGlobalSection
+EndGlobal
```
Comment (an1kh) [inline 02-Git/RockPaperScissors.sln:31]: Is this commit really needed?
If so, please include with the initial part of code

**Review by an1kh — CHANGES_REQUESTED** [review]
Please refactor (or add some commits) so the branch structure became e.g.:
1. dummy (getting choice from user) and makefile
2. random generation
3. comparison and winner determination

File: 02-Git/RockPaperScissors.sln (hunk 1 above)
Comment (SergeyChekanov) [reply 02-Git/RockPaperScissors.sln:31]: Rewrited all pullrequest, this commit isn't needed. It have to be in all commits of feature.

**Comment by SergeyChekanov** [comment]
Dead branch

**Pull request closed without merging**